.semgrep/semgrep.yml

```
@@ -0,0 +1,172 @@
+
+  # 3) Controller 계층: 허용된 HttpStatus만 사용
+  - id: controller-httpstatus-whitelist
+    languages: [java]
+    message: "허용된 HttpStatus(400/401/403/404/409)만 사용하세요."
+    severity: ERROR
+    pattern: HttpStatus.$S
+    pattern-not: HttpStatus.BAD_REQUEST
+    paths:
+      include:
+        - "domains/**/presentation/**"
+    # 다른 허용 상태 추가
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)